### DISCUSSION TITLE
How to get issue assigned?

### DISCUSSION THREAD

**Discussion by SuperGrut** [axios/axios#6635] [General]
I went through the contributing and collaboration guilde but I didn't find any mention of how to get issue assigned. I checked out few issues linked to open PR and found contributors have linked PR without being assigned to the issue. Should I go ahead and do the same?

**Comment by PritamP20** [comment]
Yeah go for it, just check once the maintainers are active or not

**Comment by jasonsaayman** [comment]
you can just assign it to yourself, ping me when you are ready for a review
